README.md

```diff
@@ -22,8 +22,12 @@ Exists in two variants:
  - `io.aiven.kafka.connect.transforms.ExtractTimestamp$Value` - works on values.
 
 The transformation defines the following configurations:
-
 - `field.name` - The name of the field which should be used as the new timestamp. Cannot be `null` or empty.
+- `timestamp.resolution` - The timestamp resolution for key or value   
+   There are two possible values: 
+    - `ms` - key or value timestamp in milliseconds
+    - `s` - key or value timestamp in seconds,  
+    default is `ms`
```

Comment:
Also, maybe full names like `seconds` and `milliseconds`? Won't bother the user much writing, but is more readable.

Reply:
Agree will change to them, more readable